### PR TITLE
feat: support vector mbtiles imports

### PR DESCRIPTION
**Pull request by achou11**
Addresses #49 

Preview (importing `countries.mbtiles` fixture from https://github.com/klokantech/vector-tiles-sample/releases/):

image

### CODE REVIEW

File: test/e2e/tilesets-import.test.js [hunk 1]
```diff
@@ -8,172 +8,220 @@ const createServer = require('../test-helpers/create-server')
...
-    'style has source pointing to correct tileset'
-  )
+    const containsSourcePointingToTileset = Object.values(style.sources).some(
+      (source) => {
+        if ('url' in source && source.url) {
+          return source.url === expectedSourceUrl
+        }
+        return false
+      }
+    )
```
Comment (gmaclennan) [inline test/e2e/tilesets-import.test.js:150]: There should be a single source (because we only imported a single source)

**Review by gmaclennan — COMMENTED** [review]
Looks good, a few comments, but I see you might have merged while I finished this. Can be follow up cleanup.

File: test/e2e/tilesets-import.test.js [hunk 2]
```diff
@@ -8,172 +8,220 @@ const createServer = require('../test-helpers/create-server')
...
+    t.equal(tilesetGetResponse.statusCode, 200)
 
-  t.same(tilesetGetResponse.json(), createdTileset)
+    const tileset = tilesetGetResponse.json()
+
+    t.same(tileset, createdTileset)
+
+    if (tileset.format === 'pbf') {
+      t.ok(
+        isValidVectorLayersValue(tileset['vector_layers']),
```
[the remark this answers is not in the document]
Comment (achou11) [reply test/e2e/tilesets-import.test.js:95]: > I think what this test really wants to test is "does the tilejson returned by map server after importing an mbtiles file match the relevant metadata that was in the metadata of the mbtiles file"

Yeah that's a good summary of what we want to test in this case

[the remark this answers is not in the document]
Comment (achou11) [reply test/e2e/tilesets-import.test.js:95]: Going to write a test for this instead. I think the mbtiles unit tests should cover this portion well enough

**Pull request merged**